### PR TITLE
Release `0.19.3`

### PR DESCRIPTION
**Pull request by ankatiyar**
## Description

Release 0.19.3 🎉 

## Developer Certificate of Origin
We need all contributions to comply with the [Developer Certificate of Origin (DCO)](https://developercertificate.org/). All commits must be signed off by including a `Signed-off-by` line in the commit message. [See our wiki for guidance](https://github.com/kedro-org/kedro/wiki/Guidelines-for-contributing-developers/).

If your PR is blocked due to unsigned commits, then you must follow the instructions under "Rebase the branch" on the GitHub Checks page for your PR. This will retroactively add the sign-off to all unsigned commits and allow the DCO check to pass.

## Checklist

- [ ] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [ ] Signed off each commit with a [Developer Certificate of Origin (DCO)](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-the-commit-signoff-policy-for-your-repository)
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [ ] Added tests to cover my changes
- [ ] Checked if this change will affect Kedro-Viz, and if so, communicated that with the Viz team

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -1,3 +1,55 @@
...
+
+## Bug fixes and other changes
+
+## Breaking changes to the API
+
+## Documentation changes
+
+## Community contributions
+
+# Upcoming Release 0.19.3
```
Comment (merelcht) [inline RELEASE.md:13]:
```suggestion
# Release 0.19.3
```

**Review by merelcht — APPROVED** [review]
🥳

File: CITATION.cff [hunk 2]
```diff
@@ -26,6 +26,8 @@ authors:
   given-names: Rashida
 - family-names: Katiyar
   given-names: Ankita
+- family-names: Ko
+  given-names: Elijah
```
Comment (noklam) [inline CITATION.cff:30]: Curious is this is following the TSC now?

**Review by noklam — APPROVED** [review]

File: CITATION.cff (hunk 2 above)
Comment (astrojuanlu) [reply CITATION.cff:30]: Yes I think we should not add new authors until they're part of the TSC

Comment (noklam) [reply CITATION.cff:30]: Meaning that we should remove Jo and Elijah from this PR

File: RELEASE.md [hunk 3]
```diff
@@ -1,3 +1,55 @@
...
+
+## Breaking changes to the API
+
+## Documentation changes
+* Enable read-the-docs search when user presses Command/Ctrl + K.
+* Added documentation for `kedro-telemetry` and the data collected by it.
+
+## Community contributions
+Many thanks to the following Kedroids for contributing PRs to this release:
+* [MosaicMan](https://github.com/MosaicMan)
```
Comment (astrojuanlu) [inline RELEASE.md:34]: + https://github.com/kedro-org/kedro/pull/3621

Comment (ankatiyar) [inline RELEASE.md:34]:
```suggestion
* [MosaicMan](https://github.com/MosaicMan)
* [Fazil](https://github.com/lordsoffallen)
```

**Review by astrojuanlu — APPROVED** [review]
🚀🔶🚀

**Pull request merged**